evm_arithmetization/src/memory/columns.rs

```diff
@@ -42,8 +42,25 @@ pub(crate) const VIRTUAL_FIRST_CHANGE: usize = SEGMENT_FIRST_CHANGE + 1;
 // Contains `next_segment * addr_changed * next_is_read`.
 pub(crate) const INITIALIZE_AUX: usize = VIRTUAL_FIRST_CHANGE + 1;
 
+// Contains the list of pruned contexts. Each pruned context must appear exactly
+// once; unused rows are set to zero.
```

Comment:
```suggestion
// and  zero if not.
```
If you want to take my suggestion